src/hip/event.jl

```diff
@@ -50,9 +50,13 @@ function synchronize(event::HIPEvent)
     return
 end
 
-function HIPEvent(stream::hipStream_t; do_record::Bool = true)
+function HIPEvent(stream::hipStream_t; do_record::Bool = true, disable_timing=true)
```

Comment:
```suggestion
function HIPEvent(stream::hipStream_t; do_record::Bool = true, timing=false)
```

Reply:
```suggestion
function HIPEvent(stream::hipStream_t; do_record::Bool = true, timing::Bool = true)
```

Reply:
@jpsamaroo you want to change the default to `timing=true`? The behavior before the PR corresponds to `timing=false`.